common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -1520,7 +1520,7 @@ def reset_problem(self, _data):
             return {
                 'success': False,
                 # Translators: 'closed' means the problem's due date has passed. You may no longer attempt to solve the problem.
-                'error': _("Problem is closed."),
+                'msg': _("Problem is closed."),
```

Comment:
Oh, is this a message that would be shown if you try to reset a problem whose due date has passed? In the test course it seems as if the Reset button stopped being shown when I changed the due date to date in the past.